lib/AFS/CellCC/Dump.pm

```diff
@@ -185,77 +185,82 @@ _calc_incremental($$) {
 # Dump the volume associated with the given job. We calculate some info about
 # the volume, dump it to disk, and report the result to the database.
 sub
-_do_dump($$$) {
-    my ($server, $job, $prev_state) = @_;
+_do_dump($$@) {
```

Comment:
the changes to `_do_dump` I think should be moved to the "multiple jobs refactoring" changes, not in here; many of the changes here are just to make `_do_dump` handle multiple jobs at once.